src/react_native/safe_area.cljs

```diff
@@ -1,32 +1,57 @@
 (ns react-native.safe-area
-  (:require
-    ["react-native-static-safe-area-insets" :default StaticSafeAreaInsets]
-    [oops.core :as oops]
-    [react-native.navigation :as navigation]
-    [react-native.platform :as platform]))
+  (:require [react-native.platform :as platform]
+            [react-native.safe-area-context :as safe-area-context]))
 
-(defn- get-static-top
-  []
-  (oops/oget StaticSafeAreaInsets "safeAreaInsetsTop"))
+(def ^:private safe-area-top-fix
+  "Subtract to the top safe area to match a standard design.
+  The values depend on our components padding (usually 12)"
+  {:android {:default 0}
+   :iphone  {62       11 ;; 16 +
+             59       12 ;; 14Pro -> 15Pro
+             :default 8}}) ;; 11 -> 14, 13mini
```

Comment:
Measured by me on simulators, only iPhone 16 was a real device, if you find this measurement wrong, please contribute to it

Reply:
Could you elaborate a little more about these numbers?

For example, are we saying that we subtract 11 units from the top safe-area on iphone 16 models?

Reply:
@seanstrom Exactly!
 
 the iPhone 16 reports a safe area of 62, if we subtract 11 units from the navigation bar, then we match the iOS Standard design (as in the image I shared & as pointed out by Pedro in the desired design). However, I noticed this number is not stable across different iPhone models :cry:  e.g. if we subtract 12 for an iPhone without dynamic island (11, 12, 13, 14), our navigation bar is positioned higher than it should be when compared to other iOS apps.
 
 Please, LMK if there are still questions about it. Also, any comment to improve this docstring is welcome